src/components/custom/dashboard/trainer/pages/plans/components/PlanCard.jsx

```
@@ -273,184 +280,190 @@ export const PlanCard = ({
-                  size="small"
-                  leftIcon={<TrashIcon size={12} />}
-                  onClick={handleDeleteClick}
-                  className="hover:bg-zinc-800 hover:text-red-400 transition-all duration-200 text-xs px-2 py-1 h-7 text-zinc-500"
-                >
-                  Delete
-                </Button>
+                    <div className="absolute right-0 top-full mt-1 bg-[#1a1a1a] border border-[#333] rounded-lg shadow-lg opacity-0 invisible group-hover:opacity-100 group-hover:visible transition-all duration-200 z-10 min-w-[120px]">
+                      <button
+                        onClick={handleCopyClick}
+                        className="w-full text-left px-3 py-2 text-xs text-gray-400 hover:text-[#FF6B00] hover:bg-[#FF6B00]/10 flex items-center gap-2"
+                      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)